src/components/Header/ListOfContents.js

```diff
@@ -1,21 +1,77 @@
+import { motion } from "framer-motion";
+
 export default function ListOfContents(props) {
+  const listVariants = {
+    visible: { transition: { staggerChildren: 0.1 } },
+  };
+
+  const itemVariants = {
+    hidden: { opacity: 0, scale: 0.5 },
+    visible: {
+      opacity: 1,
+      scale: 1,
+      transition: {
+        type: "spring",
+        damping: 15,
+        stiffness: 200,
+      },
+    },
+  };
+
   return (
-    <ul>
-      <li onClick={props.hideNav}>
-        <a href="#home">Home</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#about">About</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#skills">Skills</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#projects">Projects</a>
-      </li>
-      <li onClick={props.hideNav}>
-        <a href="#contact">Contact</a>
-      </li>
-    </ul>
+    <div>
```

Comment:
Can you please optimize this? I would create an array of objects and map it into a single component <motion.a>
```
  const linkList = [
    { href: "#home", text: "Home", scale: 1.2 },
    ....
    ....
  ];
  
```